style/CONTRIBUTE.md

```diff
@@ -248,3 +248,25 @@ Additionally, **`refsByWayName`** is an object mapping way names to text that ca
 - Each route would be recognizable by an initialism, even though it is not signposted.
 
 `refsByWayName` only works if there is no `ref` tag and the expression in the `routeConcurrency` function in style/layer/highway_shield.js includes the `name` property in the image name. The network needs to be listed as an input value that causes the `match` expression to append `name` to the image name.
+
+### Banners
+
+The shield definition supports a property **`modifiers`** which accepts an array of text strings which will be drawn atop each shield, in 10px height increments. This is used in cases where additional text is needed to differentiate shields with a common symbology, for example for [special routes of the US Numbered Highway System](https://en.wikipedia.org/wiki/List_of_special_routes_of_the_United_States_Numbered_Highway_System):
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/8/83/Business-alternate-truck_plate.svg" width=40 /><br/><img src="https://upload.wikimedia.org/wikipedia/commons/e/ec/US_30.svg" width=40 />
+
+Banners should be specified in the following cases:
+
+- When a route represents a variant of a main route with which it shares a common shield design. The banner ensures that the variant route information, which is an important component of the route, is visually displayed.
+- When two or more routes from different networks share a common symbology in the map within a common geographical area. Shields which are very similar may be drawn using common graphics for simplicity and readability, for example, when the networks differ only by a difference in text. In these cases, the most significant network should be drawn with no banner, and each of the less significant networks should be drawn with a banner.
+
+In all cases, banner text should be no more than **4** characters in length.
```

Comment:
Is this limit due to space constraints or aesthetic concerns? So far we’ve gotten lucky about finding reasonably intuitive abbreviations for standard banners, but we might run into trouble with “SCENIC”.

Reply:
4 characters is the limit after which the text has to shrink in the vertical direction to fit within 20px.  If we put in a longer string, the code would happily shrink it to fit.

Reply:
I didn't fully answer the question - this limit is due to aesthetic concerns, but we can certainly explore what 5-character banners look like to decide if this limit can/should be expanded.

Reply:
> we might run into trouble with “SCENIC”

#545 uses “SCEN”.